screens/ChatRoom/index.tsx

```diff
@@ -141,6 +143,24 @@ const ChatRoom = ({navigation, route}: ChatRoom) => {
         : chatroomDetails?.chatroom?.member?.name!
       : chatroomDetails?.chatroom?.header;
 
+  {
+    /* `{? = then}`, `{: = else}`  */
+  }
+  {
+    /* 
+          if DM ? 
+            if userID !=== chatroomWithUserID ? 
+              chatroomWithUserImageURL 
+            : memberImageURL
+          : null  
+      */
+  }
+  let chatroomProfile =
+    chatroomType === 10
+      ? user?.id !== chatroomDetails?.chatroom?.chatroom_with_user?.id
```

Comment:
store `chatroomDetails?.chatroom?.chatroom_with_user` in a variable and the use it